### PR TITLE
created contributing.md file

### LINKED ISSUE #6: Create contributing.md
### 🐣 Is your feature request related to a problem? Please describe.

Create `contributing.md` file that explains the following:

* How to open issues
* How to start working on issues
* How to make a PR

---

**Note:**
* Do mention if you're working on this issue under ` LGM SOC 2021 `.
* If you want to work on any issue, you should check if it has already been assigned to anyone.

### PR DESCRIPTION
**Pull request by coding-geek21**
## What do these changes do?
created contributing.md file

## Related Issue

Fixes #6 

## Checklist:

- [x] Have you checked there aren't other open [Pull Requests](https://github.com/Feminine-Divine/Woodog/pulls) for the same update/change?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video or GIF:

image

---

### CODE REVIEW

**Review by hariramjp777 — APPROVED** [review]

**Review by Aaishpra — CHANGES_REQUESTED** [review]
Thanks for this, can you also write a line in the readme to mention this file there, something like "please don't forget to follow contributing guidelines' and link this file.

**Comment by coding-geek21** [comment]
> Thanks for this, can you also write a line in the readme to mention this file there, something like "please don't forget to follow contributing guidelines' and link this file.

Done :)

File: README.md [hunk 1]
```diff
@@ -242,7 +242,8 @@ git checkout -b '<new-branch-name>'
 ---
 
 ## Code of conduct
-
+  
+* Please take a moment to review the [Code of Conduct](https://github.com/Feminine-Divine/Woodog/blob/master/CODE_OF_CONDUCT.md) and [contributing.md](https://github.com/Feminine-Divine/Woodog/blob/master/contributing.md) which provides the guidelines for contributing.
```
Comment (hariramjp777) [inline README.md:246]: Okay :+1:

**Review by hariramjp777 — APPROVED** [review]
Please merge this @Aaishpra

**Review by Aaishpra — APPROVED** [review]
Lgtm

**Pull request merged**